apps/systemtags/src/components/SystemTagsCreationControl.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<div id="system-tags-creation-control">
+		<h4 class="inlineblock">
+			{{ t('settings', 'System tag creation') }}
+		</h4>
+
+		<p class="settings-hint">
+			{{ t('settings', 'Allow only admins to create tags (enable or disable).') }}
```

Comment:
```suggestion
			{{ t('settings', 'If enabled, regular accounts will be restricted from creating new tags but will still be able to assign and remove them from their files.') }}
```

Weird phrasing otherwise and not proper a11y description of the following NcCheckboxRadioSwitch.
You shouldn't have to specify "enable/disable" if the main description convey the direction (Thus my suggestion: use "restrict")